src/Uno.UWP/Storage/Pickers/FileSavePicker.wasm.cs

```diff
@@ -0,0 +1,46 @@
+#if __WASM__
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.Extensions;
+using Uno.Foundation;
+using Windows.Foundation;
+using Windows.Storage.Provider;
+
+namespace Windows.Storage.Pickers
+{
+	public partial class FileSavePicker
+	{
+		public PickerLocationId SuggestedStartLocation { get; set; }
+
+		private const string LocalCachePath = "/LocalCache";
+
+		public IAsyncOperation<StorageFile> PickSaveFileAsync() => PickFilesTask().AsAsyncOperation();
+
+		private async Task<StorageFile> PickFilesTask()
+		{
+			if (!FileTypeChoices.Any())
+			{
+				throw new COMException();
+			}
+
+			if (SuggestedSaveFile == null)
+			{
+				var temporaryFolder = new StorageFolder(LocalCachePath);
+				if (!Directory.Exists(LocalCachePath))
+				{
+					temporaryFolder.MakePersistent();
```

Comment:
Why do you need this ? The existing folders from `Windows.Storage.ApplicationData.Current` may already be used to store information like this.